cfgov/v1/jinja2/v1/home_page.html

```diff
@@ -16,6 +16,12 @@
             border-bottom: 1px solid #b4b5b6;
             border-left: 1px solid #b4b5b6;
         }
+        /* Swap the homepage hero's white text to black on small screens */
```

Comment:
Maybe add a `TODO: remove inline styles when new homepage design is finalized in 2021.